### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by fristhon**
After this changes we have a more pythonic pdfplumber, I hope you found this useful

### CODE REVIEW

**Comment by jsvine** [comment]
Thanks for reformmating to conform to this project's linters. Don't worry about the codecov difference, which merely comes from reducing the total number of lines in the codebase. I'll take a look at the specific suggestions in this PR soon. I'm curious: What was the motivation for filing it?

**Comment by fristhon** [comment]
Sure, I saw pdfplumber on github trends and then decided to take a look at it.
Improving code readability and contributing to OS makes me feel good.
Thanks

File: pdfplumber/table.py [hunk 1]
```diff
@@ -143,11 +143,7 @@ def words_to_edges_v(words, word_threshold=DEFAULT_MIN_WORDS_VERTICAL):
     # Iterate through those bboxes, condensing overlapping bboxes
     condensed_bboxes = []
     for bbox in bboxes:
-        overlap = False
-        for c in condensed_bboxes:
-            if utils.get_bbox_overlap(bbox, c):
-                overlap = True
-                break
+        overlap = any(utils.get_bbox_overlap(bbox, c) for c in condensed_bboxes)
```
Comment (jsvine) [inline pdfplumber/table.py:146]: Ah, good one!

File: pdfplumber/utils.py [hunk 2]
```diff
@@ -494,7 +484,7 @@ def clip_obj(obj, bbox):
     for attr in ["x0", "top", "x1", "bottom"]:
         copy[attr] = dims[attr]
 
-    if dims["top"] != obj["bottom"] or dims["top"] != obj["bottom"]:
```
Comment (jsvine) [inline pdfplumber/utils.py:497]: I will be tweaking this further, but thank you for pointing out this redundancy.

**Review by jsvine — APPROVED** [review]
Thanks for these suggestions @fristhon. They make lots of sense, and help to make the codebase more consistent.

**Pull request merged**